src/Symfony/Component/Workflow/Exception/TransitionException.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Exception;
+
+/**
+ * @author Andrew Tch <andrew.tchircoff@gmail.com>
+ */
+class TransitionException extends \LogicException implements ExceptionInterface
```

Comment:
could it extend LogicException from the Workflow namespace? this would prevent a BC break

Reply:
Oh, so sorry - that was intended to have implement it to avoid BC. Fixed, added a simple test case also.